demisto_sdk/commands/common/tests/update_id_set_test.py

```diff
@@ -674,8 +676,8 @@ class TestPlaybooks:
             "StopScheduledTask",
         ],
         "implementing_playbooks": [
-            "Calculate Severity - Standard",
```

Comment:
Is this the edge case? Order of the items?

Reply:
yes, it tested for a subset (as set) instead of list equality.